### ISSUE TITLE
Quodlibet uses images that don't match the configured filename

### ISSUE THREAD

**Issue by dseomn** [quodlibet/quodlibet#4488]
Steps to reproduce
------------------
Configure quodlibet like:
```
[albumart]
filename = folder.png
force_filename = true
prefer_embedded = false
```

Expected Output
---------------
Quodlibet should only use files named `folder.png` as the cover image.

Actual Output
-------------
It looks like it picked an image with a different filename:
```
D: [0001.595] util.cover.manager.run: Found local cover from FilesystemCover: <_io.BufferedReader name='/home/dseomn/Music/master/Lizzo/Special/front.jpg'>
```

There's no folder.png in that directory, so it should have shown no image.

Test System
-----------

### Which version of Quod Libet?
4.6.0-1

### Which operating system 
Debian testing

### If it's audio-related, what back-end?
N/A

Additional Information
----------------------
This behavior made it harder to figure out what was going on in https://github.com/quodlibet/quodlibet/issues/3287#issuecomment-1944864059.

**Comment by dorsiflexion** [comment]
I'm actually having a look on the filesystem cover system rn. The description of this option is:

> Preferred fixed image filename(s)

So the behavior is as described. The error here is the option being named inappropriately. It should be `prefer_filename`. I don't know if we can easily rename an option though. If the option name stays, the issue might be closed.

**Comment by dseomn** [comment]
> If the option name stays, the issue might be closed.

Could it be converted to a feature request instead then? I use a single filename for cover art, and don't want quodlibet to look anywhere else.

**Comment by dorsiflexion** [comment]
What should be relatively easy is adding another boolean option which lets QL look for these names only. I could include that in my future merge request. First though, I'd like to hear from a maintainer what they think about it.

**Comment by declension** [comment]
Yes, slightly debatable perhaps, but this seems like a bug to me - "_force_ filename" should imply that QL will ignore others _even if the forced ones don't exist_. But perhaps some people expect otherwise.

I'll try to push a fix _and_ improve the comments / help around this to make it clearer.

**Comment by dorsiflexion** [comment]
@declension I could include that in my WIP merge request.

So what you're saying is the option is named correctly; and the behavior, menu description, *and* tooltip should be fixed!?

Because these three are all consistent and just *prefer* a filename, not forcing it. So I think it's just a confusing name in the config file. If you agree, should it be renamed? Can we just do that?

Regardless, should there be another toggle so QL allows the respective other behavior?

**Comment by declension** [comment]
I guess I'm agreeing with the OP @dseomn, and so yes fixing the behaviour and docstrings. I'm not sure how many people care about specifying a mild preference for files they specify over the default algorithm. Maybe this will then create change requests to the (complex) heuristics algorithm....

There's also an efficiency angle here, as a _lot_ of scanning happens unnecessarily in the use case described.
